### PR TITLE
Update README to current status

### PR DESCRIPTION
**Pull request by jinmayamashita**
## What I did

There was a difference Between roadmap and now, so I changed it.

## How to test

- [ ] ~~Is this testable with jest or e2e?~~
- [x] Does this need an update to the documentation?

Blocked by #181

### CODE REVIEW

**Comment by jinmayamashita** [comment]
Wait for #181 to be merged.

File: README.md
Comment (mvn-luanvu-hn) [inline README.md:21]: @jinmayamashita I think we should untick Storybook to implement later

File: README.md
Comment (mvn-luanvu-hn) [inline README.md:25]: We can tick after merging the UT PR

Comment (jinmayamashita) [reply README.md:25]: Now that the PR #206 has been merge, I think it can tick.
[React Hooks Testing Library](https://github.com/testing-library/react-hooks-testing-library) is not used so not ticked.

File: README.md
Comment (jinmayamashita) [reply README.md]: @mvn-luanvu-hn I agree. https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/180/commits/92a8f7cb3a9a633d8528220e09876fa046bfc0ef in done.

File: README.md
Comment (mvn-luanvu-hn) [reply README.md:25]: You're right! We do not need to use [React Hooks Testing Library](https://github.com/testing-library/react-hooks-testing-library) anymore

**Review by mvn-luanvu-hn — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**
